src/Preferences.ui

```diff
@@ -1976,10 +1976,10 @@
      <normaloff>:/icons/prefsInputAxis.png</normaloff>:/icons/prefsInputAxis.png</iconset>
    </property>
    <property name="text">
-    <string>Axis</string>
+    <string>Axes</string>
    </property>
    <property name="toolTip">
-    <string>Input driver configuration and Axis mapping</string>
+    <string>Input driver configuration and Axes mapping</string>
```

Comment:
I don't claim to be an English native speaker, but I believe on usually does not use plural in English in cases like "axis mapping" and "axis setup". The others fixes sound good though.

Reply:
I think you are right, but I am unsure as well.
If it would be button, it would clearly be button mapping and button setup.
Axis/Axes are confusing words...